frontend/src/slices/upgradesSlice.ts

```diff
@@ -1,6 +1,8 @@
 import { createSlice } from "@reduxjs/toolkit";
 import { IUpgrade } from "../../../shared/types";
 import { starterUpgrades } from "../../../shared/upgrades";
+import { resetGameData } from "./gameDataSlice";
```

Comment:
These two imports are not used

Reply:
Both imports have been deleted and taken care of.